### DISCUSSION TITLE
when will 2.12 release?

### DISCUSSION THREAD

**Discussion by liandimx** [apache/apisix#6164] [General]
i'm interest in the feature: "load balance via the client id" of plugin mqtt-proxy and want to have a try.
the example configure is stated in 2.12 doc, so i think this feature is not supported in 2.11?

**Comment by takumi913** [comment]
2.12 will released in a few days. You can subscribe the [mail list](https://lists.apache.org/list.html?dev@apisix.apache.org) for latest information

**Reply by liandimx** [reply]
thank you very much

**Reply by takumi913** [reply]
> thank you very much

Hi, @liandimx. Please close the discussion if problem has been solved.

**Reply by liandimx** [reply]
> > thank you very much
> 
> Hi, @liandimx. Please close the discussion if problem has been solved.

sorry but i can't find 'close' button?

**Reply by leslie-tsang** [reply]
@liandimx Please refer to [marking-a-comment-as-an-answer](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer)

**Reply by juzhiyuan** [reply]
Hi @liandimx, 2.12.0 just released recently, see https://github.com/apache/apisix/releases

**Comment by leslie-tsang** [comment]
Hello there, `load balance via the client id` added via #6079, it will be released in `2.12`, we are voting `2.12` in the [mail list](https://lists.apache.org/list.html?dev@apisix.apache.org), It's recommended to subscribe the mail list, more detail in [how to subscribe](https://apisix.apache.org/docs/general/subscribe-guide/)

**Reply by liandimx** [reply]
thank you. i'll try to subscribe
